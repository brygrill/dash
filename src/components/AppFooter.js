// @flow
import React, { Component } from 'react';

export default class AppFooter extends Component {
  defaultProps: {};
  state = {};
  props: {};
  render() {
    return (
      <footer className="mdl-mini-footer plata-app-footer">
        <div className="mdl-mini-footer__right-section">
          <a
            href="https://github.com/brygrill/plata-app"
            target="_blank"
            rel="noopener noreferrer"
          >
            <i className="fa fa-github fa-fw fa-2x" />{' '}
          </a>
        </div>
      </footer>
    );
  }
}
