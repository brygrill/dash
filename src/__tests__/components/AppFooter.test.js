import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import AppFooter from '../../components/AppFooter';

describe('<AppFooter />', () => {
  it('Renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<AppFooter />, div);
  });
  it('Matches snapshot with child', () => {
    const component = renderer.create(<AppFooter />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
