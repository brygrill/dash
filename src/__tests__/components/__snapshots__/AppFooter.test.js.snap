// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppFooter /> Matches snapshot with child 1`] = `
<footer
  className="mdl-mini-footer sbr-app-footer"
>
  <div
    className="mdl-mini-footer__right-section"
  >
    <a
      href="https://github.com/brygrill/sbr-training"
      rel="noopener noreferrer"
      target="_blank"
    >
      <i
        className="fa fa-github fa-fw fa-2x"
      />
       
    </a>
  </div>
</footer>
`;
